index.html

```diff
@@ -4,12 +4,580 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Homepage</title>
 </head>
 
 <body>
+    <header class="page-header">
+        <a class="main-logo"><img src="#" alt="pixmart logo">Pixmart.</a>
+        <nav class="page-nav">
+            <ul>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Products</a></li>
+                <li class="nav-item"><a href="#">Creators</a>
+                    <p class="nav-item-tag">NEW</p>
+                </li>
+                <li class="nav-item"><a href="#">Blog</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
+            </ul>
+            <div class="nav-buttons">
+                <button type="button" class="login-button">
+                    <img src="#" alt="login-logo"></button>
+                <button type="button" class="cart-button">
+                    <img src="#" alt="cart-logo"></button>
+            </div>
+        </nav>
+        <img src="#" alt="#" class="ornament-icon">

```

Comment:
The content mentioned is part of the main section, not the header, because the header is consistent across all pages.